docs/installation-and-operation/remote-sync.md

```diff
@@ -0,0 +1,347 @@
+3. Open a pull request for review.
+4. Merge the PR to production.
+5. Your **production Metabase** automatically pulls in the changes.
+
+We'll cover [setting up Remote Sync](#setting-up-remote-sync), an [example dev-to-production workflow](#an-example-dev-to-production-workflow), and [branch management](#branch-management) and some other odds and ends.
+
+Prefer the command line? Check out [serialization](./serialization.md) for command-line export and import workflows.
+
+## Setting up Remote Sync
+
+You'll need to be an admin to set up Remote Sync.
+
+1. [Set up a repository to store your content](#1-set-up-a-repository-to-store-your-content)
+2. [Create a personal access token for development](#2-create-a-personal-access-token-for-development)
+3. [Set up your development Metabase](#3-set-up-your-development-metabase)
+4. [Connect your development Metabase to your repository](#4-connect-your-development-metabase-to-your-repository)
+5. [Add an item to the Library collection](#5-add-an-item-to-the-library-collection)
+6. [Push your changes to your repository](#6-push-your-changes-to-your-repository)
+7. [Create a personal access token for production](#7-create-a-personal-access-token-for-production)
+8. [Connect your production Metabase to your repository](#8-connect-your-production-metabase-to-your-repository)
+
+## 1. Set up a repository to store your content
+
+Before you connect Metabase to your Git repository, create a [new GitHub repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository). Initialize the repo with a README.md
+
+## 2. Create a personal access token for development
+
+GitHub offers two types of personal access tokens. We recommend the fine-grained token (not the classic) because you can limit their permissions to specific repositories.
+
+1. Go to Settings > Developer settings > Personal access tokens > Fine-grained tokens.
+2. Click "Generate new token".
+3. Give it a descriptive name like "Metabase Remote Sync - Development".
+4. Select the specific repository you want Metabase to access.
+5. Add permissions:
+   - **Contents:** Read and write (it defaults to Read-only, so make sure to change this to Read and write for your development Metabase).
+   - **Metadata:** Read-only (required)
+6. Click "Generate token" and copy it immediately (you won't see it again). Store it somewhere safe; you'll need to paste it into your Metabase.
+
+For more, see GitHub's docs on [personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+
+## 3. Set up your development Metabase
+
+Remote Sync has two modes: Development and Production. 
+
+Development Metabases are where you create and edit content. You can have multiple development instances for different teams, projects, or workflows. Each connects to the same repository but can work on different branches. Development mode has:
+
```

Comment:
I feel like because we started with  "Remote Sync has two modes: Development and Production", I expected a sentence about what production mode is too. Maybe we could actually add this overview up top somewhere?